src/cc_catalog_airflow/dags/provider_api_scripts/wikimedia_commons.py

```diff
@@ -277,6 +278,20 @@ def _check_mediatype(image_info, image_mediatypes=None):
     return valid_mediatype
 
 
+def _extract_title(image_info):
+    # Titles often have 'File:filename.jpg' form
+    # We remove the 'File:' and extension from title
+    name = image_info.get('extmetadata', {}).get('ObjectName', {})
+    title = name.get('value', '')
+    if title is None:
+        title = image_info.get('title')
+    if title.startswith('File:'):
+        title = title[len('File:'):]
+    if title[-4:].lower() in ['.png', '.jpg']:
```

Comment:
Sometimes JPEG files have the .jpeg extension. Can we not slice from the rightmost dot instead?